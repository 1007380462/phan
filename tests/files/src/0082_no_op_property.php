<?php
class A {
    public $p = 42;
}
$a = new A;
$a->p;
