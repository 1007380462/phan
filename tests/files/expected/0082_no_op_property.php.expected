%s:6 NOOPError no-op property
